### PR TITLE
Lms 39+53

### PR DESCRIPTION
**Pull request by kushapusha**

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost',
+  apiUrl: 'http://192.168.2.8',
```
Comment (VictorKmin) [inline src/environments/environment.ts]: localhost

File: src/app/core/components/home-page/home-page.component.html [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <div class="home-container">
   <h1>Home Page</h1>
-  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-    Blanditiis perferendis quis veniam.
-    Atque ea et fugiat recusandae, tempore voluptates.
-    Dolor fuga harum, incidunt perspiciatis qui quod repellendus rerum voluptatem voluptates!</p>
+  <span *ngIf="authService.isAuthenticated()"
+        style="color: darkred; font-weight: bold"
```
Comment (VictorKmin) [inline src/app/core/components/home-page/home-page.component.html]: maybe better to move it to css

File: src/app/core/components/home-page/home-page.component.ts [hunk 3]
```diff
@@ -13,13 +15,32 @@ import {ErrorService} from '../../../shared';
...
+        catchError((err) =>
+          this.errorService.handleError(err)
+        )
+      )
+      .subscribe(() => {
+        if (this.userService.userInfo.subscribe()) {
+          this.userService.userInfo.subscribe(name => this.userName = name.name);
+        } else {
+          this.userService.getUserInfoByToken(this.authService.getAccessToken()).subscribe();
+          this.userService.userInfo.subscribe(name => this.userName = name.name);
```
Comment (VictorKmin) [inline src/app/core/components/home-page/home-page.component.ts]: better move this.authService.getAccessToken() to variable

File: src/environments/environment.ts (hunk 1 above)
Comment (kushapusha) [reply src/environments/environment.ts]: done

File: src/app/core/components/home-page/home-page.component.html (hunk 2 above)
Comment (kushapusha) [reply src/app/core/components/home-page/home-page.component.html]: moved

File: src/app/core/components/home-page/home-page.component.ts (hunk 3 above)
Comment (kushapusha) [reply src/app/core/components/home-page/home-page.component.ts]: done

**Pull request merged**